python/ray/air/result.py

```diff
@@ -21,12 +23,21 @@ class Result:
     Args:
         metrics: The final metrics as reported by an Trainable.
```

Comment:
If we have `checkpoint` and `best_checkpoint`, should we then also have `best_metrics` then? Or we are expecting user to just use `dataframe`?

Reply:
The latter - it is not easy to get the best results as opposed to last result from the Trial object due to how it does bookkeeping. The dataframe is how people access it right now. In the future, it would be good to improve this, though. We could have best_metric then